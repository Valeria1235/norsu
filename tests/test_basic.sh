set -v

export NORSU_PATH="$PWD/pg"

# install instance
norsu install master --extensions auto_explain

# show configure flags
"$(norsu path master)/bin/pg_config" > /dev/null

# remove only installation
rm -rf "$(norsu path master)"

# quick reinstall
norsu install master

# remove instance
norsu remove master

# remove dir
rm -rf "$NORSU_PATH"
