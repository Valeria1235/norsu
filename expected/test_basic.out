
export NORSU_PATH="$PWD/pg"

# install instance
norsu install master
Selected instance: master
	=> No work dir, choosing repo & branch
	=> Selected repo git://git.postgresql.org/git/postgresql.git
	=> Selected branch master
	=> Cloned git repo to work dir
	=> Configured sources with ['CFLAGS=-g3', '--enable-cassert']
	=> Built and installed


# show configure flags
"$(norsu path master)/bin/pg_config" > /dev/null

# remove only installation
rm -rf "$(norsu path master)"

# quick reinstall
norsu install master
Selected instance: master
	=> Current branch: master
	=> Installed build is out of date
	=> Built and installed


# remove instance
norsu remove master
Selected instance: master
	=> Removed main dir
	=> Removed work dir


# remove dir
rm -rf "$NORSU_PATH"
